testing/async_sort.cu

```diff
@@ -1,8 +1,8 @@
 #include <thrust/detail/config.h>
 
-// Disabled on MSVC for GH issue #1098
-#if THRUST_CPP_DIALECT >= 2011 && !defined(THRUST_LEGACY_GCC) && \
-  THRUST_HOST_COMPILER != THRUST_HOST_COMPILER_MSVC
+// Disabled on MSVC for GH issue #1098. Will be fixed in a future nvcc release.
+#if THRUST_CPP_DIALECT >= 2014 && \
+    THRUST_HOST_COMPILER != THRUST_HOST_COMPILER_MSVC
```

Comment:
We could add a version check here for 11.2 or greater?

Reply:
Done.